helix-core/src/test/java/org/apache/helix/TestShuffledIdealState.java

```diff
@@ -65,7 +66,8 @@ public void testInvocation() throws Exception {
     IdealCalculatorByConsistentHashing.printNodeOfflineOverhead(result3);
 
     // System.out.println(result);
-    ObjectMapper mapper = new ObjectMapper();
+    ObjectMapper mapper =
```

Comment:
Is this the only test that you picked to change?

Reply:
If an ObjectMapper needs to deserialize ZNRecord, it needs to set the introspector.
This test is the only test in helix-core that explicitly creates an ObjectMapper to work with ZNRecord. If we want to do better, this test should use the `ZNRecordSerializer` to deserialize it. To make the code minimum, I just don't want to do the refactoring work.